Remove conjurize from README (#100)
cyberark/conjur-puppet: README.md

This module provides the `conjur::secret` function, described above, and the `conjur` class, which can be configured to establish Conjur host identity on the node running Puppet.

> ### Conjur host identity with Host Factory

Reviewer: I think we should have `Conjur host identity with API key` first. I don't think HFT is more common in general than hardcoded creds per host.

If pre-establishing host identity is unfeasible, we instead recommend bootstrapping Conjur host identity using a [Host Factory](https://developer.conjur.net/reference/services/host_factory) token. Nodes inherit the permissions of the layer for which the Host Factory token was generated.

Note when used in this manner, the host factory token will only be used on the initial Puppet run, to establish identity which is then stored on the host. Subsequent runs will use that for Conjur authentication on the node side (at the time of collecting facts) and only provide the Puppet master with a temporary token to fetch the secrets with.

> To use a Host Factory token with this module, set variables `authn_login` and `host_factory_token`. Do not set the variable `authn_api_key` when using `host_factory_token`; it is not required. `authn_login` should have a `host/` prefix; the part after the slash will be used as the node’s name in Conjur.

Reviewer: You might as well fix line length in this paragraph too since you did it in other places as well

```puppet
class { 'conjur':
  account            => 'mycompany',
  appliance_url      => 'https://conjur.mycompany.com/',
  authn_login        => 'host/redis001',
  host_factory_token => Sensitive('3zt94bb200p69nanj64v9sdn1e15rjqqt12kf68x1d6gb7z33vfskx'),
  ssl_certificate    => file('/etc/conjur.pem')
}
```

> By default, all nodes using this Puppet module to bootstrap identity with host_factory_token will have the following annotation set:

Reviewer: ```suggestion
By default, all nodes using this Puppet module to bootstrap identity with `host_factory_token` will have the following annotation set:
```

```yaml
puppet: true
```

### Conjur host identity with API key

For one-off hosts or test environments it may be preferable to create a host in Conjur and then directly assign
its Conjur identity in this module.

```puppet
class { 'conjur':
  appliance_url   => 'https://conjur.mycompany.com/',
  authn_login     => 'host/redis001',
  authn_api_key   => Sensitive('f9yykd2r0dajz398rh32xz2fxp1tws1qq2baw4112n4am9x3ncqbk3'),
  ssl_certificate => file('/conjur-ca.pem')
}
```

> #### <a name="windows"></a>Using Conjur host identity with API key on Windows hosts

Reviewer: Is this link (`name=windows`) used anywhere? If not, we should remove it

Author: it's not used anywhere in this file, but I left it because I'm not sure where it ever _was_ referenced from (the docs? 🤷‍♀️). I agree I'd rather remove it, so maybe I will and worst case someone will end up in this doc and have to CTRL+F "windows"

### Conjur host identity with Host Factory

If pre-establishing host identity is unfeasible, we instead recommend bootstrapping
Conjur host identity using a [Host Factory](https://developer.conjur.net/reference/services/host_factory)
token. Nodes inherit the permissions of the layer for which the Host Factory token
was generated.

Note when used in this manner, the host factory token will only be used on the
initial Puppet run, to establish identity which is then stored on the host.

> Subsequent runs will use that for Conjur authentication on the node side (at the

Reviewer: s/node/agent/ ?

  time of collecting facts) and only provide the Puppet master with a temporary

>   token to fetch the secrets with.

Reviewer: Wrong indentation for this line and one above it

To use a Host Factory token with this module, set variables `authn_login` and
`host_factory_token`. Do not set the variable `authn_api_key` when using
`host_factory_token`; it is not required. `authn_login` should have a `host/`

> prefix; the part after the slash will be used as the node’s name in Conjur.

Reviewer: ```suggestion
prefix and the part after the slash will be used as the node’s name in Conjur.
```

### `conjur` fact

This internal, structured fact is used to inform the master of the current node
Conjur configuration and identity.

> Preestablishing configuration and identity is the recommended way of using this

Reviewer: ```suggestion
Pre-establishing configuration and identity is the recommended way of using this
```
